### PR TITLE
More detailed IANA table and new label intro

### PR DESCRIPTION
**Pull request by akeranen**

### CODE REVIEW

File: draft-ietf-core-senml.md [hunk 1]
```diff
@@ -353,6 +353,35 @@ Update Time:
   Optional. This can be used to detect the failure of sensors or
   communications path from the sensor.
 
+## SenML labels
+
+{{tbl-labels}} shows all SenML fields with their respective
+lables and data types defined by this document.
+
```
Comment (cabo) [inline draft-ietf-core-senml.md]: 1 all ... fields
2 defined by...
3 with their...

s/lables/labels/

File: draft-ietf-core-senml.md [hunk 2]
```diff
@@ -353,6 +353,35 @@ Update Time:
...
+| Data Value    | vd    |  8         | String (*) | string (*) |
+| Value Sum     | s     |  5         | Number     | double     |
+| Time          | t     |  6         | Number     | double     |
+| Update Time   | ut    |  7         | Number     | double     |
+{: #tbl-labels cols="r l r l l" title="SenML Labels"}
+
+Data Value is base64 encoded string with URL safe alphabet as defined
+in Section 5 of {{RFC4648}}, with padding omitted.
+
+For details of JSON representation see {{sec-json}}, for CBOR
```
Comment (cabo) [inline draft-ietf-core-senml.md]: of the..

**Review by cabo — APPROVED** [review]
Thanks

**Review by cabo — APPROVED** [review]

**Review by fluffy — APPROVED** [review]

**Comment by fluffy** [comment]
LGTM

**Pull request merged**